inc/gravity-forms.php

```diff
@@ -141,10 +144,15 @@ function nightingale_clean_gf_inputs( $field_content, $field ) {
 
 		// Selects.
 		case 'select':
+			// Replace li with field group
+			$field_content = str_replace( "ginput_container ginput_container_select", "ginput_container ginput_container_select nhsuk-dropdown", $field_content );
+			if ( $field->type == 'number' ) {
+				$field_content = preg_replace("#<input(.*?)class='#", "<input$1class='c-form-input ", $field_content);
+			}
 			if ( 1 === $errorflag ) {
-				$field_content = str_replace( 'gfield_select', 'nhsuk-select nhsuk-select--error', $field_content );
+				$field_content = str_replace( 'gfield_select', 'gfield_select nhsuk-select nhsuk-select--error', $field_content );
 			} else {
-				$field_content = str_replace( 'gfield_select', 'nhsuk-select', $field_content );
+				$field_content = str_replace( 'gfield_select', 'gfield_select  c-form-dropdown__select  c-form-input nhsuk-select', $field_content );
```

Comment:
does it still need the c-form classes? Think these are a hangover from nightingale 1.0 and are surplus to requirement.

Reply:
I shall double check it